Cubical/Data/Int/Properties.agda

```diff
@@ -233,6 +205,73 @@ ind-assoc _·_ f g p q base m n (suc o) =
     where
     IH = ind-assoc _·_ f g p q base m n o
 
++-comm : ∀ m n → m + n ≡ n + m
++-comm m (pos n) = ind-comm _+_ pos sucInt refl sucInt+ +sucInt pos0+ m n
++-comm m (negsuc n) = ind-comm _+_ negsuc predInt refl predInt+ +predInt negsuc0+ m n
+
 +-assoc : ∀ m n o → m + (n + o) ≡ (m + n) + o
 +-assoc m n (pos o) = ind-assoc _+_ pos sucInt +sucInt refl (λ _ _ → refl) m n o
 +-assoc m n (negsuc o) = ind-assoc _+_ negsuc predInt +predInt refl +predInt m n o
+
+-- Compose sucPathInt with itself n times. Transporting along this
+-- will be addition, transporting with it backwards will be subtraction.
+-- Use this to define _+'_ for which is easier to prove
+-- isEquiv (λ n → n +' m) since _+'_ is defined by transport
+
+-- recall
+-- sucPathInt : Int ≡ Int
+-- sucPathInt = ua (sucInt , isoToIsEquiv sucInt predInt sucPred predSuc)
```

Comment:
Shall we move definition of `sucPathInt` down here?

That means removing some of the tests above.